refextract/references/regexs.py

```diff
@@ -72,20 +72,26 @@ def compute_pos_patterns(patterns):
 
 # Pattern for arxiv numbers
 # arxiv 9910-1234v9 [physics.ins-det]
-re_arxiv = re.compile(ur"""
-    ARXIV[\s:-]*(?P<year>\d{2})-?(?P<month>\d{2})
+re_arxiv = re.compile(ur"""(?:(?:https?://(?:www\.)?arxiv\.org/(?:abs|pdf)/)|
+    (?:https?://(?:xxx\.)?lanl\.gov/(?:abs|pdf)/)|
+    (?:ARXIV[\s:-]*))(?P<year>\d{2})-?(?P<month>\d{2})
     [\s.-]*(?P<num>\d{4})(?!\d)(?:[\s-]*V(?P<version>\d))?
-    \s*(?P<suffix>\[[A-Z.-]+\])? """, re.VERBOSE | re.UNICODE | re.IGNORECASE)
+    \s*(?P<suffix>\[[A-Z.-]+\])?
+    (?:\.pdf)? """, re.VERBOSE | re.UNICODE | re.IGNORECASE)
```

Comment:
this is non-sensical  -- I wouldn't expect a `.pdf` after the category, the regexp is meant to match e.g. `arxiv:1103.01234v5 [hep-th]`, which is a common way to refer to arXiv papers.
however I don't think
`http://arxiv.org/pdf/arxiv:1103.01234v5 [hep-th].pdf` is a thing to look for